### PR TITLE
Docs/update nextjs quickstart

### PR DESCRIPTION
**Pull request by thorwebdev**
## What kind of change does this PR introduce?

- Major update of the Next.js Quickstart to utilise Auth UI, the Auth Helpers, and supabase-js V2

@dannykng Is there any way for all JavaScript / TypeScript tabs to be in sync across the page? Currently you have to switch to TypeScript manually each time as you approach each section.

### CODE REVIEW

**Comment by thorwebdev** [comment]
Reverting the example update and will open it on https://github.com/supabase/supabase/tree/master/examples instead which will be merged back into `supabase/supabase/examples` tmr.

**Comment by dannykng** [comment]
> Is there any way for all JavaScript / TypeScript tabs to be in sync across the page? Currently you have to switch to TypeScript manually each time as you approach each section.

@thorwebdev you need to add a `groupId` param to all the `<Tabs>`:

```
<Tabs
  groupId="language"
  defaultValue="js"
  values={[
    {label: 'JavaScript', value: 'js'},
    {label: 'TypeScript', value: 'ts'},
  ]}>
...
```

**Comment by thorwebdev** [comment]
Sweet, groupIds have been applied, @dannykng this is ready to merge :)

File: apps/reference/docs/guides/with-nextjs.mdx [hunk 1]
```diff
@@ -182,93 +211,148 @@ NEXT_PUBLIC_SUPABASE_URL=YOUR_SUPABASE_URL
...
+  groupId="language"
+  defaultValue="js"
+  values={[
+    {label: 'JavaScript', value: 'js'},
+    {label: 'TypeScript', value: 'ts'},
+  ]}>
+<TabItem value="js">
 
-```jsx title="components/Account.js"
+Let's create a new component for that called `Account.js` within a `components` folder.
```
Comment (dannykng) [inline apps/reference/docs/guides/with-nextjs.mdx:347]:
```suggestion
In a `components` folder, create an `Account.js` component:
```

File: apps/reference/docs/guides/with-nextjs.mdx [hunk 2]
```diff
@@ -395,63 +460,176 @@ export default function Account({ session }) {
 }
 ```
 
-### Launch!
+</TabItem>
+<TabItem value="ts">
 
-Now that we have all the components in place, let's update `pages/index.js`:
+Let's create a new component for that called `Account.tsx` within a `components` folder.
```
Comment (dannykng) [inline apps/reference/docs/guides/with-nextjs.mdx:466]:
```suggestion
In a `components` folder, create an `Account.tsx` component:
```

**Review by dannykng — APPROVED** [review]

**Pull request merged**